src/components/tabController/TabBarItem.js

```diff
@@ -47,7 +47,7 @@ export default class TabBarItem extends PureComponent {
     /**
      * icon of the tab
      */
-    icon: PropTypes.number,
+    icon: PropTypes.oneOfType([PropTypes.object, PropTypes.number]),
     /**
```

Comment:
Miki's currently working on migrating TabController to TS. 
So let's remove this one so it won't creates him conflicts..

Reply:
When could we expect this PR's changes to be merged?

Reply:
I believe next version (in two weeks or so..)